src/main/java/io/nats/client/Options.java

```diff
@@ -119,6 +119,12 @@ public class Options {
      */
     public static final Duration DEFAULT_SOCKET_WRITE_TIMEOUT = Duration.ofMinutes(1);
 
+    /**
+     * Default socket write timeout, see {@link #getSocketWriteTimeout() getSocketWriteTimeout()}.
+     * This property is defined as 1 minute

```

Comment:
I believe this doc is a copy of the field above?
Am thinking this is used as milliseconds, so that would mean this is 1 second instead.